staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -261,17 +265,29 @@ type ValidatingAdmissionPolicyBindingSpec struct {
 	// Required.
 	PolicyName string `json:"policyName,omitempty" protobuf:"bytes,1,rep,name=policyName"`
 
-	// ParamName specifies the parameter resource used to configure the admission control policy.
-	// It should point to a Custom Resource which is created out of the CRD specified in ParamSource of ValidatingAdmissionPolicy it binded.
+	// ParamRef specifies the parameter resource used to configure the admission control policy.
+	// It should point to a resource of the type specified in ParamSource of the bound ValidatingAdmissionPolicy.
 	// If the resource referred to by ParamName does not exist, this binding is considered mis-configured and the FailurePolicy of the policy definition applied.
 	// +optional
-	ParamName string `json:"paramName,omitempty" protobuf:"bytes,2,rep,name=paramName"`
+	ParamRef *ParamRef `json:"paramRef,omitempty" protobuf:"bytes,2,rep,name=paramRef"`
 
 	// MatchResources declares what resources match this binding and will be validated by it.
+	// Note that this is intersected with the policy's matchResources, so only requests that are matched by the policy can be selected by this.
+	// If this is unset, all resources matched by the policy are validated by this binding
 	// +optional
 	MatchResources *MatchResources `json:"matchResources,omitempty" protobuf:"bytes,3,rep,name=matchResources"`
```

Comment:
`//+structType=atomic` here too?

Reply:
Resolve this one to have the same conversation [above](https://github.com/kubernetes/kubernetes/pull/112883#discussion_r1002149830)